editor/icons/GuiGraphNodePort.svg

```diff
@@ -1 +1 @@
-<svg height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg"><circle cx="5" cy="5" fill="#fff" r="5"/></svg>
+<svg height="24" width="24" xmlns="http://www.w3.org/2000/svg"><circle cx="12" cy="12" fill="#fff" r="10"/></svg>
```

Comment:
`viewBox` is still required.

Reply:
Fixed. Unfortunately, my minification tool (SVG VSCode extension which uses SVGO under the hood) always removes viewBox. What do you use for SVG minification?

Reply:
SVGCleaner, https://github.com/RazrFalcon/svgcleaner. Though SVGO should have configuration options: https://github.com/svg/svgo/issues/505